src/app/components/change-referee/change-referee.component.html

```diff
@@ -0,0 +1,10 @@
+<mat-form-field>
+    <mat-label>Referee</mat-label>
+    <mat-select [(value)]="this.match.referee" (selectionChange)="setReferee(match, $event.value)">
```

Comment:
Not sure what you mean? What should do something automatically? I guess I didn't notice because VS Code does break the code anyway, but not with line breaks obviously.

Reply:
Actually, I guess it would be VSCode that automatically wraps my HTML and typescript. Maybe we should put some time into configuring the typescript linter though.

Interestingly enough I don't see that I have any extensions that would do anything regarding HTML/typescript/angular in VSCode. But something does some magic I guess :man_shrugging: